Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile
+FROM node:10-alpine
+
+# Install the dependencies in the parent folder so they don't get overriden by the bind mount
+WORKDIR /
+
+# Needed to compile some npm packages
+RUN apk add --no-cache bash python3 make g++
```

Comment:
which packages is this needed for? could we avoid them?

Reply:
`python3`, `make`, and `g++` are required for `iltorb`, a dependency of `bundlesize` and `rollup-plugin-filesize`, so it's dev only but the docker environment is meant for dev.

Reply:
ah, could we maybe have a look at updating those versions of bundlesize & rollup-plugin-filesize? if I remember it correctly in their later versions they rely on the node builtin brotli support

Reply:
Looking deeper into it only `bundlesize` is requiring iltorb, but it doesn't seem to be maintained anymore, there is a [PR](https://github.com/siddharthkp/bundlesize/pull/370) opened for 6 months, and another [package](https://github.com/siddharthkp/bundlesize2) that solves that and doesn't require any `node-gyp`.

Reply:
let's leave it as-is then for now @millotp :)